themes/default/views

```diff
@@ -0,0 +1 @@
+/var/www/vanilla/themes/default/views/
```

Comment:
That shouldn't be there. I linked a folder into the view to test something but of course I shouldn't have enclosed that...